File: Project_Files/Tests/JavaScript_SocialClub_Python_Bloodbank/model_answer/Pytest/donation.py
class Donation:
    def __init__(self, new_branch_name, new_town, new_when, new_blood_group):
        self.branch_name = new_branch_name
        self.town = new_town
        self.when = new_when
        self.blood_group = new_blood_group

    def __str__(self):
        return f'has blood group <{self.blood_group}> donated on {self.when} at {self.branch_name}'

File: Project_Files/blood_bank/MochaChai/src/Donation.js
class Donation {s
  constructor (newBranchName, newTown, newWhen, newBloodGroup) {
    // add some code here
    this.branchName = newBranchName
    this.town = newTown
    this.when = newWhen
    this.bloodGroup = newBloodGroup
  }
  toString () {
    let when = this.when.toLocaleDateString() 
    return `Has blood group <${this.bloodGroup}> donated on ${when} at ${this.branchName}`
    // return `Has blood group <${this.bloodGroup}> donated on ${when} at ${this.branchName}`
  }
}
